ERROR - 2022-03-17 17:40:35 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:40:36 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('11111111', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 11111111, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 17:42:02 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:42:03 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('11111111', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 11111111, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 17:43:38 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:43:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('11111111', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 11111111, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 17:43:46 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:43:46 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('11111111', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 11111111, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 17:44:32 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:44:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('12345678', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 12345678, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 17:47:45 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:47:45 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('12345678', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 12345678, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 17:52:08 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:52:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('12345678', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 12345678, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 17:52:34 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:52:34 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('12345678', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 12345678, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 17:55:17 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-03-17 17:55:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\alumnosRest\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\alumnosRest\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 C:\alumnosRest\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 C:\alumnosRest\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('12345678', 'numeroControl', Array, NULL)
#4 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 12345678, Array, Array)
#5 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#6 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#7 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#9 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#10 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#11 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#14 {main}
ERROR - 2022-03-17 18:08:28 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:09:21 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:15:24 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:16:02 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:17:26 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:20:36 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:21:42 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:23:50 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:28:12 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:28:20 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:28:34 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:28:50 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:29:20 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:29:25 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:33:57 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:35:48 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:35:58 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:36:32 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:48:25 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:51:33 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:52:35 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:53:14 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:54:17 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:58:10 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 18:58:36 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-03-17 18:59:06 --> syntax error, unexpected token "}", expecting "elseif" or "else" or "endif"
#0 C:\alumnosRest\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\alumnosRest\\...')
#1 C:\alumnosRest\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\alumnosRest\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\alumnosRest\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#8 {main}
CRITICAL - 2022-03-17 18:59:09 --> syntax error, unexpected token "}", expecting "elseif" or "else" or "endif"
#0 C:\alumnosRest\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\alumnosRest\\...')
#1 C:\alumnosRest\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\alumnosRest\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\alumnosRest\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#8 {main}
ERROR - 2022-03-17 18:59:26 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 19:01:15 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 19:01:49 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 19:30:19 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '.`nombre_carrera`, `especialidad`.`nombre_especialidad`, TIMESTAMPDIFF(YEAR, ...' at line 1
CRITICAL - 2022-03-17 19:30:20 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 19:30:38 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '.`nombre_carrera`, `especialidad`.`nombre_especialidad`, TIMESTAMPDIFF(YEAR, ...' at line 1
CRITICAL - 2022-03-17 19:30:38 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 19:30:56 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '.`nombre_carrera`, `especialidad`.`nombre_especialidad`
FROM `alumno`
JOIN `c...' at line 1
CRITICAL - 2022-03-17 19:30:56 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 19:32:36 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '.`nombre_carrera`, `especialidad`.`nombre_especialidad`
FROM `alumno`
JOIN `c...' at line 1
CRITICAL - 2022-03-17 19:32:37 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 19:33:23 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '.`nombre_carrera`, `especialidad`.`nombre_especialidad`
FROM `alumno`' at line 1
CRITICAL - 2022-03-17 19:33:24 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 19:34:03 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '.`nombre_carrera`, `especialidad`.`nombre_especialidad`
FROM `alumno`
WHERE `...' at line 1
CRITICAL - 2022-03-17 19:34:03 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 19:56:46 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 19:56:46 --> Call to a member function getResultArray() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:00:33 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:00:34 --> Call to a member function getResultArray() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:01:05 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:01:05 --> Call to a member function getResultArray() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:01:45 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:01:46 --> Call to a member function getResultArray() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:02:36 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:02:36 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:03:09 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:03:09 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:04:39 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:04:39 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:05:04 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:05:05 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:05:21 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:05:22 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:10:47 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:10:47 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:11:03 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:11:04 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:55:05 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:55:05 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 20:56:14 --> Unknown column 'AS' in 'field list'
CRITICAL - 2022-03-17 20:56:14 --> Call to a member function getResult() on bool
#0 C:\alumnosRest\app\Controllers\API\Alumnos.php(56): App\Models\AlumnoModel->datoAlumno('12345678')
#1 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->datosAlumnos('12345678')
#2 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#3 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#6 {main}
ERROR - 2022-03-17 21:07:38 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:08:59 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:19:08 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:21:13 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:21:41 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:21:59 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:23:27 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:26:36 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:27:44 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:35:38 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:38:31 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:40:21 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:50:29 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 21:54:54 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:08:34 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:15:12 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:18:21 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:19:35 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:19:41 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:20:46 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:27:22 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:29:52 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:30:58 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:33:35 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:38:56 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:40:02 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:46:57 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-03-17 22:47:47 --> CodeIgniter\Validation\Rules::is_unique(): Argument #2 ($field) must be of type string, null given, called in C:\alumnosRest\system\Validation\Validation.php on line 291
#0 C:\alumnosRest\system\Validation\Validation.php(291): CodeIgniter\Validation\Rules->is_unique('12345679', NULL)
#1 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 12345679, Array, Array)
#2 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#5 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#6 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#7 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#8 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#11 {main}
ERROR - 2022-03-17 22:48:46 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-03-17 22:49:53 --> CodeIgniter\Validation\Rules::is_unique(): Argument #2 ($field) must be of type string, null given, called in C:\alumnosRest\system\Validation\Validation.php on line 291
#0 C:\alumnosRest\system\Validation\Validation.php(291): CodeIgniter\Validation\Rules->is_unique('12345679', NULL)
#1 C:\alumnosRest\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('numeroControl', 'numeroControl', 12345679, Array, Array)
#2 C:\alumnosRest\system\BaseModel.php(1347): CodeIgniter\Validation\Validation->run(Array, NULL, NULL)
#3 C:\alumnosRest\system\BaseModel.php(707): CodeIgniter\BaseModel->validate(Array)
#4 C:\alumnosRest\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#5 C:\alumnosRest\app\Controllers\API\Alumnos.php(19): CodeIgniter\Model->insert(Object(stdClass))
#6 C:\alumnosRest\system\CodeIgniter.php(832): App\Controllers\API\Alumnos->create()
#7 C:\alumnosRest\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Alumnos))
#8 C:\alumnosRest\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\alumnosRest\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\alumnosRest\system\Commands\Server\rewrite.php(43): require_once('C:\\alumnosRest\\...')
#11 {main}
ERROR - 2022-03-17 22:50:10 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:51:04 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:53:43 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:55:09 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:56:12 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 22:57:16 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 23:02:42 --> Unknown column 'updated_at' in 'field list'
ERROR - 2022-03-17 23:02:48 --> Unknown column 'updated_at' in 'field list'
